File: src/test/java/uk/gov/justice/digital/hmpps/riskprofiler/config/PingEndpointTest.java
package uk.gov.justice.digital.hmpps.riskprofiler.config;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class PingEndpointTest {
    @Test
    public void ping() {
        assertThat(new PingEndpoint().ping()).isEqualTo("pong");
    }
}

File: src/test/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/config/PingEndpointTest.kt
package uk.gov.justice.digital.hmpps.riskprofiler.config

import org.assertj.core.api.Assertions
import org.junit.Test

class PingEndpointTest {
  @Test
  fun ping() {
    Assertions.assertThat(PingEndpoint().ping()).isEqualTo("pong")
  }
}
